definitions/protip/messages/money.proto

```diff
@@ -0,0 +1,183 @@
+syntax = "proto3";
+
+package protip.messages;
+
+message Money {
+  int64 amount_cents = 1;
+  enum Currency {
+    AED = 0;
+    AFN = 1;
+    ALL = 2;
+    AMD = 3;
+    ANG = 4;
+    AOA = 5;
+    ARS = 6;
+    AUD = 7;
+    AWG = 8;
+    AZN = 9;
+    BAM = 10;
+    BBD = 11;
+    BDT = 12;
+    BGN = 13;
+    BHD = 14;
+    BIF = 15;
+    BMD = 16;
+    BND = 17;
+    BOB = 18;
+    BRL = 19;
+    BSD = 20;
+    BTN = 21;
+    BWP = 22;
+    BYR = 23;
+    BZD = 24;
+    CAD = 25;
+    CDF = 26;
+    CHF = 27;
+    CLF = 28;
+    CLP = 29;
+    CNY = 30;
+    COP = 31;
+    CRC = 32;
+    CUC = 33;
+    CUP = 34;
+    CVE = 35;
+    CZK = 36;
+    DJF = 37;
+    DKK = 38;
+    DOP = 39;
+    DZD = 40;
+    EGP = 41;
+    ERN = 42;
+    ETB = 43;
+    EUR = 44;
+    FJD = 45;
+    FKP = 46;
+    GBP = 47;
+    GEL = 48;
+    GHS = 49;
+    GIP = 50;
+    GMD = 51;
+    GNF = 52;
+    GTQ = 53;
+    GYD = 54;
+    HKD = 55;
+    HNL = 56;
+    HRK = 57;
+    HTG = 58;
+    HUF = 59;
+    IDR = 60;
+    ILS = 61;
+    INR = 62;
+    IQD = 63;
+    IRR = 64;
+    ISK = 65;
+    JMD = 66;
+    JOD = 67;
+    JPY = 68;
+    KES = 69;
+    KGS = 70;
+    KHR = 71;
+    KMF = 72;
+    KPW = 73;
+    KRW = 74;
+    KWD = 75;
+    KYD = 76;
+    KZT = 77;
+    LAK = 78;
+    LBP = 79;
+    LKR = 80;
+    LRD = 81;
+    LSL = 82;
+    LTL = 83;
+    LVL = 84;
+    LYD = 85;
+    MAD = 86;
+    MDL = 87;
+    MGA = 88;
+    MKD = 89;
+    MMK = 90;
+    MNT = 91;
+    MOP = 92;
+    MRO = 93;
+    MUR = 94;
+    MVR = 95;
+    MWK = 96;
+    MXN = 97;
+    MYR = 98;
+    MZN = 99;
+    NAD = 100;
+    NGN = 101;
+    NIO = 102;
+    NOK = 103;
+    NPR = 104;
+    NZD = 105;
+    OMR = 106;
+    PAB = 107;
+    PEN = 108;
+    PGK = 109;
+    PHP = 110;
+    PKR = 111;
+    PLN = 112;
+    PYG = 113;
+    QAR = 114;
+    RON = 115;
+    RSD = 116;
+    RUB = 117;
+    RWF = 118;
+    SAR = 119;
+    SBD = 120;
+    SCR = 121;
+    SDG = 122;
+    SEK = 123;
+    SGD = 124;
+    SHP = 125;
+    SKK = 126;
+    SLL = 127;
+    SOS = 128;
+    SRD = 129;
+    SSP = 130;
+    STD = 131;
+    SVC = 132;
+    SYP = 133;
+    SZL = 134;
+    THB = 135;
+    TJS = 136;
+    TMT = 137;
+    TND = 138;
+    TOP = 139;
+    TRY = 140;
+    TTD = 141;
+    TWD = 142;
+    TZS = 143;
+    UAH = 144;
+    UGX = 145;
+    USD = 146;
+    UYU = 147;
+    UZS = 148;
+    VEF = 149;
+    VND = 150;
+    VUV = 151;
+    WST = 152;
+    XAF = 153;
+    XAG = 154;
+    XAU = 155;
+    XCD = 156;
+    XDR = 157;
+    XOF = 158;
+    XPF = 159;
+    YER = 160;
+    ZAR = 161;
+    ZMK = 162;
+    ZMW = 163;
+    BTC = 164;
+    JEP = 165;
+    EEK = 166;
+    MTL = 167;
+    TMM = 168;
+    ZWD = 169;
+    ZWL = 170;
+    ZWN = 171;
+    ZWR = 172;
+  }
+  Currency currency = 2;
```

Comment:
probably more DRY if we use a Currency *message* for this, not sure why montag duplicated them on dealer -- perhaps just laziness